Library/AppEnvironment.swift

```diff
@@ -303,13 +339,21 @@ public struct AppEnvironment: AppEnvironmentType {
     let data = userDefaults.dictionary(forKey: self.environmentStorageKey) ?? [:]
 
     var service = self.current.apiService
-    var currentUser: User?
+    var currentUser: User? // Will only be set if an OAuth token is also set
```

Comment:
Something that is a little confusing now is that you need both the `User` and the token to log in, but they're stored in different places 🤯